### PR TITLE
OFMCC-5589 - Application Location attributes

### PR DESCRIPTION
**Pull request by vietle-cgi**

### CODE REVIEW

File: frontend/src/components/facilities/FacilityLocationAttributes.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      <div>Is your facility located on Reserve?</div>
+      <v-radio-group id="reserve" v-model="model.reserve" :rules="rules.required" :hide-details="readonly" :disabled="readonly" inline color="primary">
+        <v-radio label="Yes" :value="1" />
+        <v-radio label="No" :value="0" />
+      </v-radio-group>
+    </div>
+    <div>
+      <div>
+        Is your facility designated as a
+        <a href="https://www2.gov.bc.ca/gov/content/family-social-supports/caring-for-young-children/child-care-funding/young-parent-program" target="_blank">Young Parent Program</a>
```
Comment (weskubo-cgi) [inline frontend/src/components/facilities/FacilityLocationAttributes.vue:28]: As mentioned let's make sure we get confirmation on this link. If there's some uncertainty we could move it to a environment var but that's probably overkill for a static link.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: backend/src/util/mapping/Mappings.js [hunk 2]
```diff
@@ -240,6 +240,12 @@ const FacilityMappings = [
   { back: 'statuscode', front: 'statusCode' },
   { back: '_ofm_primarycontact_value', front: 'primaryContactId' },
   { back: 'ofm_facility_review_complete', front: 'facilityReviewComplete' },
+  { back: 'ofm_on_k12_school_grounds_or_board_affiliated', front: 'k12SchoolGrounds' },
+  { back: 'ofm_in_municipal_community_center', front: 'municipalCommunity' },
+  { back: 'ofm_on_reserve', front: 'reserve' },
```
[the remark this answers is not in the document]
Comment (vietle-cgi) [reply backend/src/util/mapping/Mappings.js]: it's updated

File: frontend/src/components/applications/review/FacilityDetailsSummary.vue [hunk 3]
```diff
@@ -14,23 +14,56 @@
         </AppMissingInfoError>
       </div>
     </div>
+
+    <v-card variant="outlined" class="my-6 pa-4">
+      <div v-if="isFacilityLocationAttributesComplete(currentApplication?.facility)">
+        <v-row no-gutters>
+          <AppLabel class="mr-2">Is your facility located on K-12 school grounds or affiliated with a Board of Education?</AppLabel>
+          <div>{{ currentApplication?.facility?.k12SchoolGrounds ? 'Yes' : 'No' }}</div>
```
[the remark this answers is not in the document]
Comment (vietle-cgi) [reply frontend/src/components/applications/review/FacilityDetailsSummary.vue]: it's updated

File: frontend/src/components/facilities/FacilityLocationAttributes.vue (hunk 1 above)
Comment (vietle-cgi) [reply frontend/src/components/facilities/FacilityLocationAttributes.vue]: it's updated

File: frontend/src/components/applications/review/FacilityDetailsSummary.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (weskubo-cgi) [reply frontend/src/components/applications/review/FacilityDetailsSummary.vue]: Looks great.

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**